satellite_determination/custom_dataclasses/reservation.py

```diff
@@ -4,23 +4,28 @@
 
 from satellite_determination.custom_dataclasses.facility import Facility
 from satellite_determination.custom_dataclasses.time_window import TimeWindow
+from satellite_determination.custom_dataclasses.frequency_range import FrequencyRange
 
 
 class ReservationJsonKey(Enum):
     facility = 'facility'
     time_start = 'time_start'
     time_end = 'time_end'
+    frequency = 'frequency'
+    bandwidth = 'bandwidth'
 
 
 @dataclass
 class Reservation:
     facility: Facility
     time: TimeWindow
+    frequency: FrequencyRange
 
     @classmethod
     def from_json(cls, info: dict) -> 'Reservation':
```

Comment:
Delete this method and associated tests?

Reply:
Done